extension/src/tdigest.rs

```diff
@@ -1,8 +1,6 @@
 
 use std::{convert::TryInto, mem::take, ops::Deref};
 
-use aggregate_builder::aggregate;
```

Comment:
Line 53 still has `#[aggregate]`...

Reply:
yeah, my first attempt at reversion didn't work right. I redid it more incrementally, hopefully it'll work this time...